examples/python/aoa.py

```diff
@@ -0,0 +1,97 @@
+# Originally developed by JSBSim Team
+# Modified by Guilherme A. L. da Silva - aerothermalsolutions.co
+# Calculation required by aircraft icing enginering
+
+import jsbsim
+import xml.etree.ElementTree as ET
+import matplotlib.pyplot as plt
+import matplotlib.style as style
+style.use('fivethirtyeight')
+
+# function to change CG in aircraft xml
+# change the directory to the aircraft to be studied
+def changeCG(cgPos,readOnly):
+    tree = ET.parse('../../aircraft/global5000/global5000.xml')
+    root = tree.getroot()
+
+    for x in root.findall('mass_balance/location'):
+        cg = x.find('x').text
+        if not readOnly:
+             x.find('x').text=str(cgPos)
+             tree.write('../../aircraft/global5000/global5000.xml')
```

Comment:
```suggestion
             tree.write(os.path.join(fdm.get_root_dir(), 'aircraft/global5000/global5000.xml'))
```
Another hardcoded path. Same comment as above try not to duplicate locations where hardcoded paths are used. Using `fdm.get_root_dir()` is one way to work around this problem.